src/LevelDBStore/LevelDBStore/config.json

```diff
@@ -1,5 +1,6 @@
-﻿{
+{
   "PluginConfiguration": {
-    "Path": "Data_LevelDB_{0}"
+    "Path": "Data_LevelDB_{0}",
+    "BloomFilterBitsPerKey": 20
```

Comment:
I think this can be hardcoded.

Reply:
Maybe we should keep this modifible by user due to different hardware / node usage?

Reply:
Hardware?

Reply:
No, justing depends on data. Maybe some TPS tests are needed to make sure of this figure if it should be hard-coded instead of tunable?

Reply:
Find a best value and fix it.

Reply:
I agree with @erikzhang.
@Qiao-Jin, I believe it may be possible to even adjust it during running time in order to help the search for the best. You can slightly keep changing and tracking it during execution. For example:

* Increase in 1 unit
* Decrease in 1 unit
* Increase in 5 units
* Decrease in 5 units
* Increase in 10 units
* Decrease in 10 units

This is a like a Variable Neighborhood Search.
You slightly modify the bloom filter and observe the results, if not good you revert.
Basically you need an average value and some observation after change.

On the real code I believe that it is not possible because the microbenchmark is costly.

Reply:
@Tommo-L If we resolve this conversation we can merge, the code looks good to me

Reply:
Okay, we'll add the benchmark with it.

Reply:
After testing it seems that bitsPeyKey = 15 is a good value to choose, with highest TPS.